src/libostree/ostree-repo.h

```diff
@@ -1180,6 +1191,16 @@ gboolean ostree_repo_traverse_commit_union_with_parents (OstreeRepo         *rep
                                                          GCancellable       *cancellable,
                                                          GError            **error);
 
+_OSTREE_PUBLIC
+gboolean ostree_repo_traverse_commit_with_flags (OstreeRepo                     *repo,
```

Comment:
New API also needs to be listed in `libostree-devel.sym`.

I'm not sure how this patch can work without that actually.